### ISSUE TITLE
PNC children are not appearing at all

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw#1460]
In the 27 Oct Chad release pointing to wcaro stage, I'm noticing that there are NO children appearing in any of the PNC profiles.

Steps to replicate:

- Register ANC woman
- Fill out pregnancy outcome form.
- Put outcome is live birth and register a child
- Go to the new PNC profile
- The child that was just registered is not appearing.

**Comment by qaziabubakar-vd** [comment]
@cafootitt I looked into this issue. I followed the steps to replicate it and for the delivery date, while filling out the pregnancy outcome form, I selected a date one month prior to today i.e. 28th Sep 2020 and when I opened the PNC profile, I couldn't see the baby listed there.
What's happening is that there is a condition that for the baby to appear in PNC forms the date of birth (delivery date) should be 28 days prior to the current date, at the most, so if the baby is born 29 days (or more) prior to the current date then it's filtered out and does not appear in the PNC profile. 
Current condition = `date_of_delivery + 28 days >= Current Date`

Kindly let me know if this is the intended functionality or should I update the logic on delivery date.
